.github/release.yaml

```
@@ -0,0 +1,18 @@
+---
+
+# https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
+
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking-change
+    - title: New Features 🎉
+      labels:
+        - enhancement
+    - title: Other Changes
+      labels:
+        - "*"
```

If it is what I think, we can align this with what GCG produce with the tags we have setup for consistency:

```suggestion
  categories:
    - title: Breaking changes
      labels:
        - backwards-incompatible
    - title: Implemented enhancements
      labels:
        - enhancement
    - title: Fixed bugs
      labels:
        - bug
    - title: Merged pull requests
      labels:
        - "*"
```